pkg/identity/numericidentity.go

```diff
@@ -284,6 +286,13 @@ var (
 	ErrNotUserIdentity = errors.New("not a user reserved identity")
 )
 
+// UpdateReservedIdentitiesMetrics updates identity metrics based on the reserved identities.
+func UpdateReservedIdentitiesMetrics() {
+	for range reservedIdentities {
```

Comment:
small nit, you can do `metrics.IdentityCount.Add(len(reservedIdentities))`

Reply:
@aanm Fixed :smile: